watertap/unit_models/electrodialysis_1D.py

```diff
@@ -1556,6 +1691,165 @@ def eq_dl_thickness(self, mem, side, t, x):
                     * self.current_dens_lim_x[t, x] ** -1
                 )
 
+    def _get_fluid_dimensionless_quantities(self):
+        self.general_mass_diffusivity = Var(
```

Comment:
Should this be diffus_phase_comp in prop package?

Reply:
Ah, "not individual ions". Maybe change naming to line up more closely with `diffus_phase_comp`?

Reply:
done.